Tests/LoggerTests/LoggerPerformanceTests.swift

```diff
@@ -180,3 +180,16 @@ class LoggerPerformanceTests: XCTestCase {
         }
     }
 }
+
+// MARK: - LogManager + asyncWait
+
+private extension LogManager {
+    /// !!! This method only serves for unit tests !!! Before checking values (XCT checks), unit tests must wait for loging jobs to complete.
+    /// Loging is being executed on a different queue (logingQueue) and thus here the main queue waits (sync) until all of logingQueue jobs are completed.
+    /// Then it executes the block within logingQueue.sync which is empty, so it continues on doing other things.
+    func waitForLogingJobsToFinish() {
+        serialLoggingQueue.sync {
+            //
```

Comment:
Nechybí tu něco?

Reply:
Cekal bych ze ne, perdpokladam ze cilem je, aby se vyprazdnila serialLoggingQueue a tim ze chces synchrone vykonat nejaky blok kodu (i prazdny), tak toho docilis.

Reply:
jak pise David. To casem zmizi, az to nahradime nejakou injectovanou queue asi, ale zatim to musi bejt takhle...